config/settings.yml

```diff
@@ -9,6 +9,11 @@
   :openstack_network:
     :is_admin: false
     :inventory_object_refresh: true
+    :allow_targeted_refresh: true
+  :cinder:
+    :is_admin: false
```

Comment:
just a quick quiestion, the :all_tenants param doesn't work for storage and network?

Reply:
It does, but they're an option the defaults to being off because at some point someone told me we supported refresh with a non-admin user so I was trying to work around that.